extension/src/lib.rs

```diff
@@ -210,7 +224,9 @@ enum VersionedStateRead {
 
 /// Wrapper to enable versioning of runtime calls.
 #[derive(Decode, Debug)]
-enum VersionedDispatch<T: Config> {
+enum VersionedDispatch<
+	T: frame_system::Config<RuntimeCall: GetDispatchInfo + Dispatchable<PostInfo = PostDispatchInfo>>,
```

Comment:
Why cant this just take an opaque RuntimeCall as a generic parameter? It doesnt need anything more than that - e.g. the whole Runtime type being passed in only to use the RuntimeCall from it. This is private, so the calling function can just specifiy T::RuntimeCall, keeping this implementation simple.

Reply:
Agree with you, pushed an update to convert it to only receive RuntimeCall instead of a whole Config. This part is a missing part during the refactor of the devnet extension. Thanks for pointing it out.